### DISCUSSION TITLE
Rich Snippet Google Test not detected  by URL(putting fragment of code, validates correctly)

### DISCUSSION THREAD

**Discussion by eybel** [vercel/community#1543] [Help]
Hi there!

This is really weird but google rich snippet test tool by URL on a live server it wont detect the schema at all, it wont come up. Not even the script. It seems that the code its not present on the <head> tag unless you hit "view page source" So google cant read it...

Using the <Script... component didnt help, it wont appear not even no the "view page source" option

```
    <script
      type="application/ld+json"
      dangerouslySetInnerHTML={{ __html: JSON.stringify(schemaObject) }}
      key="product-jsonld"
    />
```

I am using getStaticProps with getStaticPaths

Somebody from google may have said that it's not there when request ed( related to the rendering delays? )

I have noticed that using the console, the script with the schema is not present but its present if ti RIGHT CLICK mouse and hit "View Page Code". I. copy the whole HTML page, everything and it passes the validation with not even 1 warning, everything seems eprfect in two places "schema.org" and "google rich snippet test tool".

Its definitely not available by default unless you hit "view page source"

The Schema is totally fine but when using the URL to scan I get "No items detected"

**Comment by amyegan** [comment]
Hi Abel! Are you able to share a minimal reproduction so we can get a better look at what is going wrong?

**Reply by eybel** [reply]
> Hi Abel! Are you able to share a minimal reproduction so we can get a better look at what is going wrong?

Definitly. Check it out here:

https://stage.tableclothslat.com/neutral-earth-tone-beige-lace-rectangle-placemat-13-inch-x-20-inch

or any products from here:
https://stage.tableclothslat.com/store

Results:
https://search.google.com/test/rich-results/result?id=AH6WZHYzQ7-HK3bsx1YcRg

Screenshot:
https://ibb.co/TLL3dtX

If you "view source code" and copy the whole page code (from <Html> to </Html>, then paste it in the google test as a fragment.,you get everything validated:
https://ibb.co/cyCDLff
https://ibb.co/5vZtnz5

I have tried the <Script /> component as well...

At this moment my code:
https://ibb.co/jD8nWXX
https://ibb.co/0GYVkxC

**Reply by eybel** [reply]
> Hi Abel! Are you able to share a minimal reproduction so we can get a better look at what is going wrong?

Is there any updates on this?
